### PR TITLE
Update the Javadoc URL

### PR DESCRIPTION
**Pull request by brfrn169**
This PR updates the Javadoc URL to https://javadoc.io/doc/com.scalar-labs/scalardb/latest/index.html and removes the original Javadoc files.

Please take a look!

### CODE REVIEW

**Review by Torch3333 — DISMISSED** [review]
Thank you, LGTM

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ dependencies {
     * [Making Cassandra more capable, faster, and more reliable](https://www.slideshare.net/scalar-inc/making-cassandra-more-capable-faster-and-more-reliable-at-apacheconhome-2020) at ApacheCon@Home 2020
     * [Scalar DB: A library that makes non-ACID databases ACID-compliant](https://www.slideshare.net/scalar-inc/scalar-db-a-library-that-makes-nonacid-databases-acidcompliant) at Database Lounge Tokyo #6 2020
     * [Transaction Management on Cassandra](https://www.slideshare.net/scalar-inc/transaction-management-on-cassandra) at Next Generation Cassandra Conference / ApacheCon NA 2019
-* [Javadoc](https://scalar-labs.github.io/scalardb/javadoc/)
+* [Javadoc](https://javadoc.io/doc/com.scalar-labs/scalardb/latest/index.html)
```
Comment (feeblefakie) [inline README.md:24]: I think we can/should also refer scalardb-rpc and scalardb-server?

I manually typed scalardb-rpc and scalardb-server, and they don't exist but it seems to trigger creation.
Screen Shot 2021-08-06 at 10 34 04

**Review by feeblefakie — COMMENTED** [review]
Thank you! I just noticed that it shows only core API.

File: README.md (hunk 1 above)
Comment (brfrn169) [reply README.md:24]: Sure. I will update it. Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**